sources/packages/web/src/types/contracts/Common.ts

```diff
@@ -1,12 +1,23 @@
-// TODO: When all icon are replace to font awesome,
-// replace  icon?: string | string[] with
-// icon?:string[];
+import { RouteLocationRaw } from "vue-router";
+
+// todo: use MenuInterface and once MenuModel removed from all, delete the MenuModel
```

Comment:
This be marked as deprecated to be easier to identify in the code also.
![image](https://user-images.githubusercontent.com/61259237/190219273-27ee097e-ede7-483a-b011-621d69ddc08a.png)